.github/ISSUE_TEMPLATE/enhancement-request.yaml

```
@@ -0,0 +1,89 @@
+name: "Security docs enhancement request"
+description: Submit an issue to the security documentation team to request an improvement to the Elastic Security documentation.
+title: "[enhancement]: "
+labels: "enhancement"
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Hello! Use this form to submit an enhancement request to the [@elastic/security-docs](https://github.com/orgs/elastic/teams/security-docs) team. 
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: Summarize your request here. What is the problem?
+      placeholder: |
+        What: We need documentation for X.
+        Why: We've received support requests that indicate customers are struggling with Y in versions A and B. 
+    validations:
+      required: true
+  - type: textarea
+    id: related
+    attributes:
+      label: Related links / assets
+      description: How can we learn more about the problem?
+      value: |
+        Please include each of the following, if applicable:
+        Subject matter expert:
+        Figma link(s):
+        Github epic link(s):
+        Github issue link(s):
+    validations:
+      required: false
+  - type: dropdown
+    id: doc-set
+    attributes:
+      label: Which documentation set needs improvement?
+      description: Stateful, Serverless, or both?
+      options:
+        - Stateful and Serverless
+        - Stateful only
+        - Serverless only
+        - Unknown / Not applicable
+      default: 0
+    validations:
+      required: true
+  - type: textarea
+    id: doc-set-differences
+    attributes:
+      label: Feature differences
+      description: If you selected both Stateful and Serverless above, please describe how, if at all, the feature differs in each deployment method.
+      placeholder: The feature is identical in Stateful and Serverless.
+    validations:
+      required: true
+  - type: textarea
+    id: software-version
+    attributes:
+      label: Software version
+      description: Is the problem applicable to a particular release version? If so, which one(s)?
+      placeholder: |
+        For example:
+        "The functionality was introduced in 8.11.0, so docs need updating for 8.11+"; 
+        "The docs are good after version 8.10, but need updating in 8.7, 8.8, and 8.9"
+    validations:
+      required: true
+  - type: textarea
+    id: collaborators
+    attributes:
+      label: Collaborators
+      description: If applicable, please include contact information for the responsible product manager, designer, and developer.
```

Nit: "responsible" on its own sounds like, are these trustworthy and mature people, not are they _responsible for_ a specific feature.
```suggestion
      description: If applicable, please include contact information for the relevant product manager, designer, and developer.
```